react-native/react/menubar/index.render.desktop.js

```diff
@@ -46,48 +85,14 @@ export default class Render extends Component {
     const {openKBFS, openKBFSPublic, openKBFSPrivate, showMain, showHelp, quit, openingButtonInfo} = this.props
 
     return (
-      <div style={{backgroundColor: colors.white, ...commonStyles.fontRegular, display: 'flex', flexDirection: 'column', height: '100%'}}>
-        <Header openKBFS={openKBFS} showHelp={showHelp}/>
-        {this.props.openingMessage && <OpeningMessage message={this.props.openingMessage} buttonInfo={openingButtonInfo} showHelp={showHelp}/>}
-        {this.props.username && <FolderList username={this.props.username} openKBFSPublic={openKBFSPublic} openKBFSPrivate={openKBFSPrivate} folders={this.props.folders}/>}
-        <Footer debug={this.props.debug || false} showHelp={showHelp} quit={quit} showMain={showMain}/>
-      </div>
-    )
-  }
-}
-
-class Header extends Component {
-  props: {
-    openKBFS: () => void,
-    showHelp: () => void
-  };
-
-  render () {
-    return (
-      <div style={{display: 'flex', flexDirection: 'row', paddingTop: 11, paddingLeft: 11, paddingBottom: 8}}>
```

Comment:
We have globalStyles for things like flexBox so we shouldn't put the stuff in manually